packages/SwingSet/src/kernel/dynamicVat.js

```diff
@@ -104,13 +127,21 @@ export function makeDynamicVatCreator(stuff) {
     }
 
     function makeErrorResponse(error) {
+      if (!notifyNewVat) {
```

Comment:
If the consequence of this flag being `false` here is a kernel panic, I think probably the flag should have a different name.

Reply:
fair point, I think I have a way to rewrite that flow: instead of conditionals in the callbacks, I'll do something like this at the bottom of the file:

```js
    const p = Promise.resolve().then(build);
    if (notifyNewVat) {
      p.then(makeSuccessResponse, makeErrorResponse)
        .then(sendResponse)
        .catch(err => console.error(`error in createVatDynamically`, err));
    } else {
      p.catch(errorDuringReplay);
    }
```